workflows/edit_eml/edit_spatialvectors.Rmd

```diff
@@ -1,3 +1,65 @@
 ## Edit spatialVectors
 
-Occasionally, you may encounter a third type of data object: `spatialVector`. This object contains spatial data, such as a shapefile or geodatabase. Editing a `spatialVector` is similar to editing a `dataTable` or an `otherEntity`. An `attributeList` and `physical` should also be present for a `spatialVector`.
\ No newline at end of file
+Occasionally, you may encounter a third type of data object: `spatialVector`. This object contains spatial data, such as a shapefile or geodatabase. Editing a `spatialVector` is similar to editing a `dataTable` or an `otherEntity`. A `physical` and `attributeList` should be present. One important difference is that a `spatialVector` object should also have a `geometry` slot that describes the geometry features of the data. The possible values include one or more (in a list) of 'Point', 'LineString', 'LinearRing', 'Polygon', 'MultiPoint', 'MultiLineString', 'MultiPolygon', or 'MultiGeometry'.
+
+To add a `geometry` slot use:
+
+```{r, eval = FALSE}
+eml@dataset@spatialVector[[1]]@geometry[[1]] <- read_eml("<geometry>Polygon</geometry>")
+```
+
+Additionally, spatial data should typically be archived within a compressed .zip file in order to reduce file sizes. Also note that shapefiles limit attribute names to 10 characters, so attribute names in the metadata may not match exactly to attribute names in the data.
```

Comment:
we really archive these as .zips to ensure that the files stay together. the different types of geospatial files (.prj, for example) are both nearly useless on their own and frequently are required to be kept in the same directory for geospatial programs. zipping helps make sure nothing gets lost, and that when a user downloads the file they get all of the files

Reply:
Ah, right! I'll change that.